### PR TITLE
add description for MCMO

### PR DESCRIPTION
**Pull request by liuchunchi**
MCMO- multi client multi operator, more general situation

### CODE REVIEW

File: draft-liu-nasr-architecture.md [hunk 1]
```diff
@@ -101,38 +101,76 @@ This process is repeated periodically to continuously assure compliance.
...
+|                   |     +------------+-----+-----------------------+----+----------------+    |                   |
+|    Client  A      |                  |     |                       |    |                     |    Client  B      |
+|                   |  Path            |     |                       |    |         Path        |                   |
+|  +--------------+ |  Request         |     |                       |    |         Attestation |   +------------+  |
+|  |  Relying     +-+----------+ +-----+-----+-------+        +------+----+-------+ Result (PAR)|   |  Relying   |  |
+|  |  Party       | |          | |     |     |       |        |      |    |       |     +-------+---+  Party     |  |
+|  +---+----------+<+--------+ | |     |     |       |  Intra |      |    |       |     |       |   +------------+  |
+|      |            |        | | |  RE |     |AR     |  ISP   |   RE |    | AR    |     |       |           ^       |
+|      |            | Answer | +-> +---+-----v-----+ |  API   |  +---+----v----+  |     |       |           |       |
+|      |Path        | Report |   | | Orchestrator  +-+--------+->| Orchestrator|<-+-----+       |           |       |
```
Comment (Maisy-ML) [inline draft-liu-nasr-architecture.md:120]: Usually, response is used more than answer, respond report

File: draft-liu-nasr-architecture.md [hunk 2]
```diff
@@ -101,38 +101,76 @@ This process is repeated periodically to continuously assure compliance.
...
+                          |               |                             |                  |                                 
+                          +---------------+-----------------------------+------------------+                                 
+                                          |  Verifier software/hardware |                                                    
++-------------------+                     |  Reference Value            |                       +-------------------+        
+|                   |            +--------+----------+        +---------+---------+             |                   |        
+|                   |            |        |          |        |         |         |             |                   |        
+|    Client  A      |            |  +-----v------+   |        |   +-----v------+  |             |    Client  B      |        
+|                   |  Path      |  | Verifier   |   |        |   |  Verifier  |  | Path        |                   |        
+|  +--------------+ |  Request   |  | Vendor A   |   |        |   |  Vendor B  |  | Attestation |   +------------+  |
```
Comment (Maisy-ML) [inline draft-liu-nasr-architecture.md:155]: I think "verifier from vendor A" is better than verifier vendor A

**Review by Maisy-ML — COMMENTED** [review]
I have reviewed and have two comments

File: draft-liu-nasr-architecture.md (hunk 1 above)
Comment (liuchunchi) [reply draft-liu-nasr-architecture.md:120]: It is actually answer/report, but i get your point. 
I will just leave "report"

File: draft-liu-nasr-architecture.md (hunk 2 above)
Comment (liuchunchi) [reply draft-liu-nasr-architecture.md:155]: OK

**Comment by liuchunchi** [comment]
@Maisy-ML Revised.

**Pull request merged**